src/vorta/views/utils.py

```diff
@@ -13,3 +18,31 @@ def get_colored_icon(icon_name):
     svg_img = QImage.fromData(svg_str).scaledToHeight(128)
 
     return QIcon(QPixmap(svg_img))
+
+
+def process_log(self, context):
+    cmd = context.get('cmd')
+    if cmd is not None and cmd != 'init':
+        msgid = context.get('msgid')
+        repo_url = context.get('repo_url')
+        if msgid == 'LockTimeout':
+            profile = BackupProfileModel.get(name=context['profile_name'])
+            msg = QMessageBox()
+            msg.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+            msg.setParent(self, QtCore.Qt.Sheet)
+            msg.setText(
+                translate(
+                    "MainWindow QMessagebox",
+                    f"The repository at {repo_url} might be in use by another computer. Continue?"))
```

Comment:
```suggestion
                    f"The repository at {repo_url} might be in use by another computer. Continue?"))
```
"Continue?" is kind of vague, but I don't know how else to word it.

Reply:
Maybe "Continue anyways?". I don't want to get into the technical details about break-lock.

Reply:
"Continue anyways" sounds more scary.

Reply:
The repository at {repo_url} might be in use by another computer. Override it and continue?

Reply:
Also good wording.